README.md

```diff
@@ -846,6 +846,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1600+ peo
   * [loader.io](https://loader.io/) — Free load testing tools with limitations
   * [meercode.io](https://meercode.io/) — Meercode is the ultimate monitoring dashboard for your CI/CD builds. Free for open-source and one private repository.
   * [netdata.cloud](https://www.netdata.cloud/) — Netdata is an open-source tool to collect real-time metrics. It's a growing product and can also be found on GitHub!
+  * [API Hat](https://apihat.com) — API management & Uptime monitoring, Free plan: 50 Uptime monitor, 15 heartbeat monitors, 3 Domain monitor and 3 Statuspage free.
```

Comment:
there isn't clear pricing information, it says 'free while in beta' and no details of eventual pricing, so I can't verify what you say is true.  Can you link a page with this details?